cime_config/machines/config_machines.xml

```diff
@@ -227,7 +227,7 @@
 
       <modules compiler="amdclang">
         <command name="load">PrgEnv-aocc</command>
-        <command name="load">aocc/3.2.0</command>
+        <command name="load">aocc/4.0.0</command>
```

Comment:
Was this change required for the new version of ADIOS (2.9.1)?

Reply:
It was required for amdclang compiler to build on pm-cpu (3.2.0 module is no longer available).